aws/resource_aws_codebuild_project_test.go

```diff
@@ -290,6 +291,18 @@ func testAccCheckAWSCodeBuildProjectDestroy(s *terraform.State) error {
 }
 
 func testAccAWSCodeBuildProjectConfig_basic(rName, vpcConfig, vpcResources string) string {
+	// This is used for testing aws_codebuild_webhook as well as aws_codebuild_project.
+	// In order for that resource to work the Terraform AWS user must have done a GitHub
+	// OAuth dance.
+	//
+	// Additionally, the GitHub user that the Terraform AWS user logs in as must have
+	// access to the GitHub repository. This allows others to run tests for the webhook
+	// without having to have access to the Packer GitHub repository.
+	sourceURL, ok := os.LookupEnv("CODEBUILD_GITHUB_SOURCE_URL")
```

Comment:
This is a great idea -- I'll make sure this gets ported over in some fashion when I rebase this PR.